### ISSUE TITLE
Fix error with Karma test runner

### ISSUE THREAD

**Issue by jedeen** [excaliburjs/Excalibur#891]
### Context
Running `npm run all` and `npm run tests` throws an error:

```
Error: Could not find source map for: "E:\Projects\Excalibur\build\dist\excalibur.js"
  at CoverageTransformer.addFileCoverage (E:\Projects\Excalibur\node_modules\remap-istanbul\lib\CoverageTransformer.js:148:17)
  at E:\Projects\Excalibur\node_modules\remap-istanbul\lib\CoverageTransformer.js:268:14
  at Array.forEach (native)
  at CoverageTransformer.addCoverage (E:\Projects\Excalibur\node_modules\remap-istanbul\lib\CoverageTransformer.js:266:24)
  at E:\Projects\Excalibur\node_modules\remap-istanbul\lib\remap.js:48:13
  at Array.forEach (native)
  at Reporter.remap (E:\Projects\Excalibur\node_modules\remap-istanbul\lib\remap.js:47:16)
  at E:\Projects\Excalibur\node_modules\karma-typescript\dist\karma\reporter.js:38:42
  at Array.forEach (native)
  at Collection.forEach (E:\Projects\Excalibur\node_modules\karma\lib\browser_collection.js:93:21)
  at onRunComplete (E:\Projects\Excalibur\node_modules\karma-typescript\dist\karma\reporter.js:26:26)
  at Server.<anonymous> (E:\Projects\Excalibur\node_modules\karma\lib\events.js:13:22)
  at emitTwo (events.js:111:20)
  at Server.emit (events.js:191:7)
  at emitRunCompleteIfAllBrowsersDone (E:\Projects\Excalibur\node_modules\karma\lib\server.js:295:12)
  at Server.<anonymous> (E:\Projects\Excalibur\node_modules\karma\lib\server.js:326:7)
  at emitOne (events.js:96:13)
  at Server.emit (events.js:188:7)
  at Server.<anonymous> (E:\Projects\Excalibur\node_modules\karma\lib\server.js:309:12)
  at emitTwo (events.js:111:20)
  at Server.emit (events.js:191:7)
  at Browser.onComplete (E:\Projects\Excalibur\node_modules\karma\lib\browser.js:150:13)
  at Socket.<anonymous> (E:\Projects\Excalibur\node_modules\karma\lib\events.js:13:22)
  at emitTwo (events.js:111:20)
  at Socket.emit (events.js:191:7)
  at E:\Projects\Excalibur\node_modules\socket.io\lib\socket.js:503:12
  at _combinedTickCallback (internal/process/next_tick.js:73:7)
  at process._tickCallback (internal/process/next_tick.js:104:9)
```

Note: This error does not fail the build

### Proposal
Prevent this error from occurring. Investigate Karma fix or property change to prevent Karma from throwing this error. Alternatively, adjust how our compilation works to prevent this error.

**Comment by alanag13** [comment]
I have not seen this since we moved to webpack.

**Comment by eonarheim** [comment]
This was fixed in the #970 webpack update
